sw/device/tests/aes_entropy_test.c

```diff
@@ -60,9 +60,6 @@ OTTF_DEFINE_TEST_CONFIG();
 bool test_main(void) {
   dif_aes_t aes;
 
-  // First of all, we need to get the entropy complex up and running.
```

Comment:
The entropy cfg is needed so that AES receives PRNG values to clear data in/out/key values, done on lines 116-117. The PRNG uses CSRNG bits from the entropy source as seed. 

I wonder if the checks on line 121 and 127 should be improved further (the output / keys should not be all 1s or all 0s either, apart from not being equal to the previous values). That way, we at least know they were indeed garbage.

Reply:
Thanks for the review @sriyerg, I'm going to improve the lines 121 and 127 as you suggested.
I just have two questions about the entropy_src.

1. Are you happy with the removal of this entropy_src initialization from the other aes tests?
2. Regarding the `aes_entropy_test.c`, isn't the initialization done [here](https://github.com/lowRISC/opentitan/blob/c0a96f9b46e96aa83a2db0469383067c36fcb5e8/sw/device/lib/testing/test_rom/test_rom_start.S#L185) by the `test_rom` enough?

Reply:
Ohh, I did not realize it was already called by the test_rom. In that case, LGTM.